api/PerformanceResourceTiming.json

```diff
@@ -1000,9 +1008,21 @@
             "web-features:resource-timing"
           ],
           "support": {
-            "chrome": {
-              "version_added": "43"
-            },
+            "chrome": [
+              {
+                "version_added": "133"
+              },
+              {
+                "version_added": "115",
+                "partial_implementation": true,
+                "notes": "`responseStart` used interim response instead of final response ([chromestatus](https://chromestatus.com/feature/5158830722514944))."
+              },
+              {
+                "version_added": "85",
+                "partial_implementation": true,
+                "notes": "Full implementation but without explicit support for interim response."
```

Comment:
What was the observed behavior of `responseStart` in Chrome < 115? What timing did it return?

Reply:
As it does after 133. Basically 115 - 132 was a temporary blip where we tried something else and have now gone back.

Annoyingly we can’t say it was fully supported before 115 and fully supported in and after 133 as BCD insists on only one fully supported version 😔

Reply:
I was a bit confused, but took a look now: After adding `version_removed` on the last two statemnets, I was able to remove the partial flag (see 17e4a9e2f).